kge/util/dump.py

```diff
@@ -113,90 +113,174 @@ def _add_dump_trace_parser(subparsers_dump):
     parser_dump_trace = subparsers_dump.add_parser(
         "trace",
         help=(
-            "Process and dump trace to stdout and/or csv. The trace will be processed "
-            "backwards, starting with a specified job_id."
+            "Dump trace to csv (default) and/or stdout. The tracefile will be processed"
+            " backwards starting from the last entry. Further options allow to start"
+            " processing from a particular checkpoint or job_id or epoch_number."
         ),
     )
-
     parser_dump_trace.add_argument(
         "source",
         help="A path to either a checkpoint or a job folder.",
         nargs="?",
         default=".",
     )
-
+    parser_dump_trace.add_argument(
+        "--train",
+        action="store_const",
+        const=True,
+        default=False,
+        help=(
+            "Include entries from training jobs (enabled when none of --train, --valid,"
+            " or --test is specified)."
+        ),
+    )
+    parser_dump_trace.add_argument(
+        "--valid",
+        action="store_const",
+        const=True,
+        default=False,
+        help=(
+            "Include entries from validation during training and evaluation on"
```

Comment:
From validation or evaluation jobs on the valid split

Reply:
It also includes validation entries when one validates on the training data. That's why I wrote it like that.

Reply:
Yes, but it's redundant. Compare "evaluation on the valid split" and "validation and evaluation on the valid split"